src/sandstorm/run-bundle.c++

```diff
@@ -1183,18 +1189,17 @@ private:
     KJ_SYSCALL(sigprocmask(SIG_SETMASK, &sigset, nullptr));
   }
 
-  void copyEtc() {
+  void linkEtc() {
     auto files = splitLines(readAll("etc.list"));
```

Comment:
If /etc/resolv.conf is a symlink to /etc.host/resolv.conf and
/etc.host/resolv.conf is a symlink to /etc/resolvconf/run/resolv.conf and
/etc/resolvconf/run/resolv.conf is a symlink to
/etc.host/resolvconf/run/resolv.conf , then I think we're golden.

Since we control /etc/ we can make the above happen, I believe.
​
The simplest hard-coded way to do this (IMHO) is to add
/etc/resolvconf/run/resolv.conf to the whitelist, and all will magically
Just Work. If /etc.host/resolvconf/run/resolv.conf doesn't exist, then we
create a broken symlink, but that's not a huge deal.